osu.Framework/Graphics/UserInterface/Dropdown.cs

```diff
@@ -63,7 +63,7 @@ private void setItems(IEnumerable<T> items)
             foreach (var entry in items)
                 addDropdownItem(GenerateItemText(entry), entry);
 
-            if (Current.Value == null || !itemMap.Keys.Contains(Current.Value))
+            if (Current.Value == null || !itemMap.Any(i => EqualityComparer<T>.Default.Equals(i.Key, Current.Value)))
```

Comment:
You could also write

```suggestion
            if (Current.Value == null || !itemMap.Keys.Contains(Current.Value, EqualityComparer<T>.Default))
```

It seems to read a little better to me but I'm not fussed.